src-gui/src/renderer/components/modal/introduction/IntroductionModal.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+    makeStyles,
+    Modal,
+} from '@material-ui/core'
+import { useState } from 'react'
+import Slide01_GettingStarted from './slides/Slide01_GettingStarted'
+import Slide02_ChooseAMaker from './slides/Slide02_ChooseAMaker'
+import Slide03_PrepareSwap from './slides/Slide03_PrepareSwap'
+import Slide04_ExecuteSwap from './slides/Slide04_ExecuteSwap'
+import Slide06_ReachOut from './slides/Slide06_ReachOut'
+import Slide05_KeepAnEyeOnYourSwaps from './slides/Slide05_KeepAnEyeOnYourSwaps'
+import { setFetchFiatPrices, setUserHasSeenIntroduction } from 'store/features/settingsSlice'
+import { useAppDispatch, useSettings } from 'store/hooks'
+import Slide00_FiatPricePreference from './slides/Slide00_FiatPricePreference'
+
+const useStyles = makeStyles({
+    modal: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+    paper: {
+        width: '80%',
+        display: 'flex',
+        justifyContent: 'space-between',
+    },
+})
+
+export default function IntroductionModal() {
+    const userHasSeenIntroduction = useSettings((s) => s.userHasSeenIntroduction);
+
+    const dispatch = useAppDispatch();
+
+    // Handle Display State
+    const [open, setOpen] = useState<boolean>(!userHasSeenIntroduction)
+    const [showFiat, setShowFiat] = useState<boolean>(true)
```

Comment:
Instead of having another local state in the component here you can:
```react
// Use the selector to fetch the fiat price status directly from the global Redux state
const fetchFiatPrices = useSettings((s) => s.fetchFiatPrices);

// Dispatch a Redux action when the toggle is switched (onChange)
dispatch(setFetchFiatPrices(showFiat))
```

You can also put this directly into `Slide00_FiatPricePreference.tsx` which avoids having to pass the callback function into the component and unnecessary state in `IntroductionModal.tsx`